composer.json

```diff
@@ -63,6 +63,11 @@
     "extra": {
         "composer-exit-on-patch-failure": true,
         "drupal-scaffold": {
+            "file-mapping": {
+                "[project-root]/.gitignore": false,
+                "[web-root]/INSTALL.txt": false,
```

Comment:
Explicitly exclude both INSTALL and README as these files have been reported to be a source of identification of the Drupal version (on misconfigured servers), that opens an additional attack vector.